src/support/archive.cpp

```diff
@@ -210,8 +210,8 @@ static uint32_t read32be(const uint8_t* buf) {
 }
 
 void Archive::dump() const {
-  printf("Archive data %p len %zu, firstRegularData %p\n", data.data(),
-         data.size(), firstRegularData);
+	printf("Archive data %p len %zu, firstRegularData %p\n", data.data(),
```

Comment:
We had talked before about making a `.clang-format` file but I think it hasn't been checked in yet. Also the files in Binaryen now come from several sources, so the formatting isn't in good shape at all right now... :(